variables.tf

```diff
@@ -144,3 +144,9 @@ variable "cloudflare_zone_id" {
   type        = string
   default     = null
 }
+
+variale "security_policy" {
```

Comment:
typo

Reply:
where is the type? 
it's security_policy

Reply:
oh variable